feature/bgp/routereflector/README.md

```diff
@@ -0,0 +1,44 @@
+# RT-1.8: BGP Route Reflector Test
+## Summary
+*   BGP route reflector capabilities check
+*   Ensure functionality of different OC paths for "supported-capabilities", "BGP peer-type", "BGP Neighbor details" and "BGP transport session parameters"
+
+## Topology
+ATE (Port1) <-EBGP-> (Port1) DUT (Port2) <-IS-IS/IBGP-> (Port2) ATE, DUT (Port3) <-IS-IS/IBGP-> (Port3) ATE
+*   Connect ATE Port1 to DUT port1 (EBGP peering)
+*   Connect ATE Port2 to DUT port2 (For IS-IS adjacency and IBGP peer reachaility)
+*   Connect ATE Port3 to DUT port3 (For IS-IS adjacency and IBGP peer reachaility)
+
+## Procedure
+*   Establish IS-IS adjacency between ATE Port2 <-> DUT Port2, ATE Port3 <-> DUT Port3.
+*   Establish BGP sessions as follows between ATE and DUT.
+  *  ATE port 2 and ATE port3 are emulating RR clients peered with the DUT acting as the RR server. DUT's loopback address should be used  for the IBGP peering and "transport/config/local-address" OC path should be used on DUT to configure BGP transport address for IBGP peering. ATE addresses used for the IBGP peering (different from ATE Port1 and ATE Port2 addreses) and DUT loopback addresses should be reachable via IS-IS. Each of the IBGP peering should advertise 5M IPv4 and 2M IPv6 prefixes. Among these, 2M v4 and 1M v6 prefixes are unique. Among the 2M v4 prefixes as well, 1.5M are non RFC1918 or RFC6598 addresses and represent Internet prefixes.
+  *   The DUT Port1 has eBGP peering with ATE Port 1 and is receiving 10M IPv4 and 3M IPv6 routes. These simulate Internet routes. Out of these, only 1.5M v4 and 600k IPv6 prefixes are unique and will be programmed to the FIB.
```

Comment:
Which direction are the prefixes being advertised?  (I am guessing from DUT to ATE?)

Are the 1.5M 'Internet' prefixes the same 1.5M prefixes advertised from DUT to ATE the same 1.5M 'Internet' prefixes learned from the iBGP RRC's?

Reply:
I added more context and details ptal. Mapping w/ a real world scenario

Reply:
ok, this is now clear to me for the eBGP prefixes: "Out of these, only 1.5M v4 and 600k IPv6 prefixes are unique and will be programmed to the FIB"